napari_arboretum/visualisation/matplotlib_plotter.py

```diff
@@ -23,8 +23,6 @@ def draw_current_time_line(self, time: int) -> None:
         if not hasattr(self, "_mpl_time_line"):
             self._mpl_time_line: Line2D = self.axes.axvline(time, color="white")
         else:
-            # TODO: fix this
```

Comment:
Did this get fixed? Not sure what it is

Reply:
Yeah, it's an old comment I forgot to delete in a previous PR, sorry for the confusion!